### ISSUE TITLE
Use environment variable in execute(...) bind action

### ISSUE THREAD

**Issue by D630** [junegunn/fzf#481]
Hey!

It would be nice, if we could specify the shell in `execute()` via an environment variable (`FZF_SHELL || SHELL`).  

For example, it would be easier to use functions, when sh is not my interactive shell like on Debian (dash and bash resp.):

```sh
% echo $SHELL
/bin/bash
% readlink -m $(command -v sh)
/bin/dash

# Use FZF_SHELL
% function __me { echo I am: "$1" ; }
% typeset -f -x __me 
% FZF_SHELL=$SHELL fzf --reverse --expect=enter --bind="enter:execute(__me {})"

# instead of executing bash in dash
% fzf --reverse --expect=enter --bind="enter:execute[exec bash -c '__me () { echo I am: {} ; } ; __me']"
% fzf --reverse --expect=enter --bind="enter:execute[exec bash -c 'source FILE-WITH-FUNC ; __me']"
% fzf --reverse --expect=enter --bind="enter:execute[exec FILE-WITH-BASH-CODE]"
```

**Comment by junegunn** [comment]
I like the idea. It would be better if we don't have to introduce another env var and just rely on `$SHELL`. But it only points to the default shell, and does not change when you switch between different shells. Do you think using `$SHELL` is enough for the purpose?

```sh
jg@junegunn:~> echo $SHELL
/bin/bash
jg@junegunn:~> zsh
junegunn% echo $SHELL
/bin/bash
junegunn% fish
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
jg@junegunn ~> echo $SHELL
/bin/bash
```

**Comment by netei** [comment]
I think using SHELL is enough, people should be using the default shell feature if they really intend to use a particular shell.

**Comment by junegunn** [comment]
@netei Thanks for the feedback. I just realize that we can still override `SHELL` if we want to, so no need for `FZF_SHELL`

```sh
SHELL=zsh fzf --bind 'enter:execute:echo $ZSH_VERSION; sleep 1'
```

**Comment by junegunn** [comment]
Pushed the update. Let me know if there's a better approach though.
The change will be included in the next version which I'm thinking of releasing this weekend.
I appreciate your suggestions.

**Comment by D630** [comment]
Ok, thanks :)

**Comment by D630** [comment]
Will FZF_DEFAULT_COMMAND follow SHELL as well?

**Comment by junegunn** [comment]
@D630 Good point. We should change that too, right?

**Comment by D630** [comment]
We should ;)

Thank you!

**Comment by junegunn** [comment]
Released 0.11.3 with the change. Thanks.

**Comment by netei** [comment]
> @D630 Good point. We should change that too, right?

Yes, I think it should follow it too :-). Thanks for the new release !
